pkg/dmaasbackup/schedule.go

```diff
@@ -69,15 +72,10 @@ func (d *dmaasBackup) generateBackupName(dbkp v1alpha1.DMaaSBackup) string {
 	return dbkp.Name
 }
 
-// updateScheduleInfo checks for relevant velero schedule/backup and
-// update the dbkp with schedule/backup details
-func (d *dmaasBackup) updateScheduleInfo(dbkp *v1alpha1.DMaaSBackup) error {
-	// TODO
-	return nil
-}
-
-// cleanupOldSchedule remove the old schedules if fullbackup retention mentioned
 func (d *dmaasBackup) cleanupOldSchedule(dbkp *v1alpha1.DMaaSBackup) error {
-	// TODO
-	return nil
+	if dbkp.Spec.PeriodicFullBackupCfg.CronTime == "" {
```

Comment:
this will be empty in case of restic, but, retention count will be set

Reply:
yes. Handling for restic is targeted in another PR.